### ISSUE TITLE
View Rec Sites and Trails in a list

### ISSUE THREAD

**Issue by BAAlexK** [bcgov/nr-rec-resources#148]
#### Description:

**As a** *public visitor* **I want** *to be able to view a list of recreation sites and trails* **So That** *I can find the RST I am interested in*

#### Acceptance Criteria:
- [x] View basic information
   - [x] Include more detailed information such as activities, open/closed status
     - [x] #204 Activities sub-issue
     - [x] #205 site status sub-issue
- [x] Pagination of view (load more)
- [x] Default to Alphabetical order
- [x] using real data with new RSt schema

#### Dependencies
- Blocked by migrating live data
- related to #204 #205 

#### Notes:
- Current iteration of parks.ca doesn't have pagination in their search
- Need server side pagination (if use pagination) as volume impacts complexity
- Limit list count shown on each page?
- Need to write API

**Comment by JLWade** [comment]
moving this one to done :)
Filters are handled in another ticket
